changelog/2024-11-08-conduit-0-12-2-release.mdx

```diff
@@ -0,0 +1,34 @@
+---
+slug: '2024-11-08-conduit-0-12-2-release'
+title: Conduit v0.12.2 release
+draft: false
+tags: [conduit, performance]
+---
+
+v0.12.2 comes with up to 5x more throughput, new CLI command for initializing
+Conduit and pipelines, as well as bug fixes.
+
+<!--truncate-->
+
+### Key Highlights
+
+- **Up to 5x higher throughput**: We started revamping the internal engine
+  that's running pipelines which resulted in a 5x higher throughput. If you're
+  interested in how we did it, you can find more information
+  in [the pull request](https://github.com/ConduitIO/conduit/pull/1913). We'll
+  also publish a blog post where we will go into the details. The feature is
+  still experimental, and can be enabled with the `-preview.pipeline-arch-v2`
```

Comment:
Here i would include how running the entire command to better illustrate it:

```bash
$ conduit -preview.pipeline-arch-v2
```

I'm also wondering if we should have this buried in a changelog or we should highlight it elsewhere (even while it's experimental). Somewhere we could link people to. What do you think @hariso ?

Reply:
We should also include this flag in `/docs/configuration`.

Reply:
I have added the full command and updated the configuration page, good callout! 

As for highlighting the performance stuff: given that it's a config param affecting performance, we can either mention it in the configuration page (which we just did) or on a performance optimization page (which we currently don't have). Given that it's still experimental, I think it's not bad that we leave it on the configuration page.